rfc/rfc006-distributed-registry.md

```diff
@@ -261,7 +261,14 @@ If a DID Document doesn't have any other controllers, and the control over the p
 
 ### 3.4 Privacy considerations
 
-All data is public knowledge. All considerations from [§10 of did-core](https://www.w3.org/TR/did-core/#privacy-considerations) apply.
+The Nuts network is a mesh network without central authority. Each node in the network will have a copy of every DID Document published on the network.
```

Comment:
mention published DID documents can't be deleted, so extra care should be taken regarding what is published?